xarray/tests/test_backends_datatree.py

```diff
@@ -396,15 +396,12 @@ def test_phony_dims_warning(self, tmpdir) -> None:
                 }
 
 
-@pytest.mark.skipif(
-    have_zarr_v3, reason="datatree support for zarr 3 is not implemented yet"
-)
-@requires_zarr
+@requires_zarr_v3
```

Comment:
Should this just be `@requires_zarr`? Does this mean we aren't testing for zarr2 anymore?

Reply:
I fixed this in https://github.com/aladinor/xarray/pull/1/commits/c2a1f5fb39788683f1a5df2776e12e44f289595e, which ensures we test for both versions of zarr-python and all supported `zarr_formats`.